README.md

```diff
@@ -9,3 +9,17 @@ Run the build script from the root of the repo and follow the prompts:
 ```
 
 Once you are ready to use the new image for an application, go to the repo for that application and update the `FROM` statement in the Dockerfile to point at the new version of the image.
+
+## Notable container changelog
+
+### Rails 2.6.3
+
+* `rails-base:2.6.3_20190429154415`: Initial base container.
+  * _Circle: `rails-base:2.6.3-circleci_20190501132020`_
+
+### Rails 3.0.2
+
+* `rails-base:3.0.2_20210812134927`: Initial base container.
```

Comment:
everything else

Reply:
Your comments seem like the different rails versions are used for different repos, does that mean this isn't a change from 2.6.3 -> 3.0.2 but are instead two separate rails base containers running simultaneously?

Mostly just wondering if I'm understanding correctly why they both have the comment "Initial Base Container" if we changed from one to the next.

Reply:
it's the initial 3.0.2 container, not related to 2.6.3 container at all. and then within ruby versions we'll iterate.

so, specifically with base containers:

core api and workers are still on rails-base:2.6.3_20190429154415
utilization api is on rails-base:3.0.2_20240501155324
everything else is on rails-base:3.0.2_20210812134927

so this should be useful for two reasons: 1) knowing which container is the newest one for each version (like if we start up a new rails app, we'll know to build it off rails-base:3.0.2_20240501155324) and 2) have a better idea of what the container is when looking at an app (for example, looking at utilization api, we see it uses rails-base:3.0.2_20240501155324 and then we can go to this changelog and see that the new thing there was adding jemalloc).